etc/kayobe/pulp.yml

```diff
@@ -428,6 +428,15 @@ stackhpc_pulp_images_kolla:
   - redis-base
   - redis-sentinel
 
+# Whitespace-separated list of regular expressions matching Kolla image names.
```

Comment:
White space separated matches what we have in https://github.com/stackhpc/stackhpc-release-train/pull/121/files. Not related to this PR, as none of the image names contain anyway whitespace, but I had the sudden realisation that this means that the short names for the repositories can't have any whitespace in.